test/mocha/helpers.js

```diff
@@ -166,6 +166,7 @@ api.buildHistory = async ({historyId, mockData, nodes, witnesses} = {}) => {
   // set workers to use passed witnesses when merging
   for(const key in nodes) {
     const ledgerNode = nodes[key];
+    // double check to ensure witnesses is a Set
```

Comment:
Is this comment relevant?

Reply:
this is definitely my bad and I have removed this comment here: https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/pull/248/commits/e2b620c0750bebb9164eaa56f10625a59e85e2f2